src/themes/default/components/core/blocks/MyAccount/MyOrder.vue

```diff
@@ -137,13 +143,36 @@
 </template>
 
 <script>
+import Vue from 'vue'
 import MyOrder from '@vue-storefront/core/compatibility/components/blocks/MyAccount/MyOrder'
 import ReturnIcon from 'theme/components/core/blocks/Header/ReturnIcon'
+import ProductImage from 'theme/components/core/ProductImage'
+import { getThumbnailPath } from '@vue-storefront/core/helpers'
+import { mapActions } from 'vuex'
 
 export default {
   mixins: [MyOrder],
   components: {
-    ReturnIcon
+    ReturnIcon,
+    ProductImage
+  },
+  data () {
+    return {
+      itemThumbnail: []
+    }
+  },
+  methods: {
+    ...mapActions({
+      getProduct: 'product/single'
+    })
+  },
+  beforeMount () {
+    this.skipGrouped(this.order.items).forEach(async item => {
```

Comment:
could you also with this PR change `skipGrouped` method to computed `singleOrderItems` or something like this? It should really be a computed property not a method, and then here (and in other places) it would just be `this.singleOrderItems.find(...`

Reply:
I don't really see what do you mean by using find. We process a whole array here, we're not aiming at getting a single element, could you elaborate?